<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="mainHBoxLayout">
      <item>
       <layout class="QVBoxLayout" name="leftVBoxLayout">
        <item>
         <widget class="QTabWidget" name="leftTabWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Communication</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="communicationVBoxLayout"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Settings</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="settingsVBoxLayout">
              <item>
               <layout class="QFormLayout" name="settingsFormLayout">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Camera Device</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="cameraDeviceSpinBox"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Camera Resolution</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QComboBox" name="cameraResolutionComboBox">
                  <property name="editable">
                   <bool>false</bool>
                  </property>
                  <item>
                   <property name="text">
                    <string>640x480</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>1920x1080</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Frame Rate (ms)</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="frameRateLineEdit"/>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Marker Size (m)</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="markerSizeLineEdit"/>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Arena Width (m)</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QLineEdit" name="arenaWidthLineEdit"/>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Arena Height (m)</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QLineEdit" name="arenaHeightLineEdit"/>
                </item>
                <item row="6" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Calibration File</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1">
                 <widget class="QLineEdit" name="calibrationFileLineEdit"/>
                </item>
                <item row="7" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="8" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Camera Brightness</string>
                  </property>
                 </widget>
                </item>
                <item row="8" column="1">
                 <widget class="QSlider" name="cameraBrightnessSlider">
                  <property name="maximum">
                   <number>255</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="9" column="0">
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Camera Sharpness</string>
                  </property>
                 </widget>
                </item>
                <item row="9" column="1">
                 <widget class="QSlider" name="cameraSharpnessSlider">
                  <property name="maximum">
                   <number>255</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="10" column="0">
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>Camera Focus</string>
                  </property>
                 </widget>
                </item>
                <item row="10" column="1">
                 <widget class="QSlider" name="cameraFocusSlider">
                  <property name="maximum">
                   <number>255</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="7" column="1">
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="calibrationFilePushButton">
                    <property name="text">
                     <string>Browse</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="Line" name="line">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="settingsButtonsHBoxLayout">
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="applySettingsPushButton">
                  <property name="text">
                   <string>Apply</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="rightVBoxLayout">
        <item>
         <widget class="QLabel" name="imageLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
